ni_measurementlink_service/measurement/service.py

```diff
@@ -128,18 +130,35 @@ class MeasurementService:
 
     """
 
-    def __init__(self, measurement_info: MeasurementInfo, service_info: ServiceInfo) -> None:
-        """Initialize the Measurement Service object with measurement info and service info.
+    def __init__( self, service_config_path: str, version: str, ui_file_paths: List[str]) -> None:
```

Comment:
Extra space (ni-python-styleguide also complained)
```suggestion
    def __init__(self, service_config_path: str, version: str, ui_file_paths: List[str]) -> None:
```